addfunction.js

```diff
@@ -0,0 +1,354 @@
+// let a10=document.createElement('a')
+// let img1=document.createElement('img')
+// /*second project*/ 
+// let div8=document.createElement('div')
+// let article1=document.createElement('article')
+// let div9=document.createElement('div')
+// let article2=document.createElement('article')
+// let div10=document.createElement('div')
+// let div11=document.createElement('div')
+// let div12=document.createElement('div')
+// let a11=document.createElement('a')
+// let img2=document.createElement('img')
+// /*third project*/
+// let div13=document.createElement('div')
+// let article3=document.createElement('article')
+// let div14=document.createElement('div')
+// let article4=document.createElement('article')
+// let div15=document.createElement('div')
+// let div16=document.createElement('div')
+// let div17=document.createElement('div')
+// let a12=document.createElement('a')
+// let img3=document.createElement('img')
+// /*fourth project*/
+// let div18=document.createElement('div')
+// let article5=document.createElement('article')
+// let div19=document.createElement('div')
+// let article6=document.createElement('article')
+// let div20=document.createElement('div')
+// let div21=document.createElement('div')
+// let div22=document.createElement('div')
+// let a13=document.createElement('a')
+// let img4=document.createElement('img')
+// /*fifth project*/
+// let div23=document.createElement('div')
+// let article7=document.createElement('article')
+// let div24=document.createElement('div')
+// let article8=document.createElement('article')
+// let div25=document.createElement('div')
+// let div26=document.createElement('div')
+// let div27=document.createElement('div')
+// let a14=document.createElement('a')
+// let img5=document.createElement('img')
+// /*sixth project*/
+// let div28=document.createElement('div')
+// let article9=document.createElement('article')
+// let div29=document.createElement('div')
+// let article10=document.createElement('article')
+// let div30=document.createElement('div')
+// let div31=document.createElement('div')
+// let div32=document.createElement('div')
+// let a15=document.createElement('a')
+// let img6=document.createElement('img')
+
+function mobile_menu() {
+    body.append(section);
+    section.append(i0);
+    section.append(ul);
+    ul.append(li0, li1, li2, li3)
+    li0.append(a0);
+    li1.append(a1);
+    li2.append(a2);
+    li3.append(a3);
+
+    section.setAttribute('class', 'Menubutton')
+    i0.setAttribute('class', 'material-icons')
+    ul.setAttribute('class', 'menu_ul')
+    a0.setAttribute('href', '#hello');
+    a0.setAttribute('onclick','section.remove()')
+    a1.setAttribute('href', '#portfolio');
+    a1.setAttribute('onclick','section.remove()')
+    a2.setAttribute('href', '#about');
+    a2.setAttribute('onclick','section.remove()')
+    a3.setAttribute('href', '#contact');
+    a3.setAttribute('onclick','section.remove()')
+
+    i0.textContent='close';
```

Comment:
- [ ] OPTIONAL
Your button does not appear to be a button when a user hover over it. Kindly improve the UX of the close button by making it a pointer, by including the CSS property: `cursor: pointer` to the close button